### ISSUE TITLE
CustomEdible custom coordinates are not optional and server hangs if they are not specified

### ISSUE THREAD

**Issue by mmarquez76** [OmnifactoryDevs/DimensionalEdibles#33]
#29 seems to have caused a new issue. If I disable the `S:customCoords` field, it is automatically rewritten with an empty field. Then, when players try to use a Void World Cake, if they don't already have a cached spawn location, the server hangs until it terminates from a single tick taking longer than 60 seconds.

I'm trying to make it so that players get sent to the corresponding coordinate in the void world when they use the cake, as opposed to everyone being sent to the same place, which isn't desirable on my particular server.

The config gets reset to this if the `S:customCoords` field is commented out:
```
            # Set a list of custom coordinates used by Custom Cakes / Apples, this is optional.
            # Format: <Dimension ID>, <X>, <Y>, <Z>
            # Example: 0, 420, 123, -420
            S:customCoords <
             >
```

The crash report is pasted below:

```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// Uh... Did I do that?

Time: 2023-01-31 09:29:03 EST
Description: Watching Server

java.lang.Error: ServerHangWatchdog detected that a single server tick took 60.00 seconds (should be max 0.05)
    at net.minecraft.world.World.getWorldType(World.java:3208)
    at net.minecraft.world.chunk.Chunk.getBlockState(Chunk.java:472)
    at net.minecraft.world.chunk.Chunk.getBlockState(Chunk.java:467)
    at net.minecraft.world.World.getBlockState(World.java:912)
    at jackyy.dimensionaledibles.util.TeleporterHandler.scanColumn(TeleporterHandler.java:182)
    at jackyy.dimensionaledibles.util.TeleporterHandler.getValidYSpawnPos(TeleporterHandler.java:163)
    at jackyy.dimensionaledibles.util.TeleporterHandler.getDimPos(TeleporterHandler.java:150)
    at jackyy.dimensionaledibles.block.BlockCakeBase.calculateCoordinates(BlockCakeBase.java:304)
    at jackyy.dimensionaledibles.block.BlockCakeBase.teleportPlayer(BlockCakeBase.java:294)
    at jackyy.dimensionaledibles.block.BlockCakeBase.onBlockActivated(BlockCakeBase.java:126)
    at jackyy.dimensionaledibles.block.BlockCustomCake.onBlockActivated(BlockCustomCake.java:84)
    at net.minecraft.server.management.PlayerInteractionManager.processRightClickBlock(PlayerInteractionManager.java:449)
    at net.minecraft.network.NetHandlerPlayServer.processTryUseItemOnBlock(NetHandlerPlayServer.java:741)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:55)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:11)
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13)
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
    at java.util.concurrent.FutureTask.run(Unknown Source)
    at net.minecraft.util.Util.runTask(SourceFile:529)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723)
    at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Thread Dump --
  Threads: "pool-4-thread-4" Id=66 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "pool-4-thread-3" Id=65 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "pool-4-thread-2" Id=64 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "pool-4-thread-1" Id=63 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@17534a4c
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "Chunk I/O Executor Thread-1" Id=62 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1994b6d6
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@1994b6d6
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "threadDeathWatcher-2-1" Id=58 TIMED_WAITING
           at java.lang.Thread.sleep(Native Method)
           at io.netty.util.ThreadDeathWatcher$Watcher.run(ThreadDeathWatcher.java:152)
           at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
           at java.lang.Thread.run(Unknown Source)
           
           
           "Netty Server IO #1" Id=57 BLOCKED on java.util.ArrayDeque@1dab213c owned by "Server thread" Id=18
           at net.minecraft.server.MinecraftServer.func_175586_a(MinecraftServer.java:1373)
           -  blocked on java.util.ArrayDeque@1dab213c
           at net.minecraft.server.MinecraftServer.func_152344_a(MinecraftServer.java:1395)
           at net.minecraft.world.WorldServer.func_152344_a(WorldServer.java:1335)
           at net.minecraft.network.PacketThreadUtil.func_180031_a(SourceFile:10)
           at net.minecraft.network.NetHandlerPlayServer.func_147347_a(NetHandlerPlayServer.java:454)
           at net.minecraft.network.play.client.CPacketPlayer.func_148833_a(SourceFile:126)
           at net.minecraft.network.play.client.CPacketPlayer$Rotation.func_148833_a(SourceFile:88)
           at net.minecraft.network.NetworkManager.channelRead0(NetworkManager.java:147)
           ...
           
           
           "File IO Thread" Id=56 TIMED_WAITING
           at java.lang.Thread.sleep(Native Method)
           at net.minecraft.world.storage.ThreadedFileIOBase.func_75736_b(SourceFile:51)
           at net.minecraft.world.storage.ThreadedFileIOBase.run(SourceFile:30)
           at java.lang.Thread.run(Unknown Source)
           
           
           "spark-forge-async-worker" Id=55 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@58a9bced
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@58a9bced
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
           at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           ...
           
           
           "spark-cpu-monitor" Id=54 TIMED_WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@55258c9e
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@55258c9e
           at java.util.concurrent.locks.LockSupport.parkNanos(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(Unknown Source)
           at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
           at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           ...
           
           
           "pool-3-thread-1" Id=53 WAITING on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@693ba2ea
           at sun.misc.Unsafe.park(Native Method)
           -  waiting on java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject@693ba2ea
           at java.util.concurrent.locks.LockSupport.park(Unknown Source)
           at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(Unknown Source)
           at java.util.concurrent.LinkedBlockingQueue.take(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.getTask(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
           at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
           at java.lang.Thread.run(Unknown Source)
           
           
           "Server Watchdog" Id=52 RUNNABLE
           at sun.management.ThreadImpl.dumpThreads0(Native Method)
           at sun.management.ThreadImpl.dumpAllThreads(Unknown Source)
           at net.minecraft.server.dedicated.ServerHangWatchdog.run(ServerHangWatchdog.java:45)
           at java.lang.Thread.run(Unknown Source)
           
           
           "Ender IO Scheduler" Id=50 TIMED_WAITING
           at java.lang.Thread.sleep(Native Method)
           at crazypants.enderio.base.scheduler.Scheduler.run(Scheduler.java:41)
           
           
           "Netty Server IO #0" Id=43 RUNNABLE (in native)
           at sun.nio.ch.WindowsSelectorImpl$SubSelector.poll0(Native Method)
           at sun.nio.ch.WindowsSelectorImpl$SubSelector.poll(Unknown Source)
           at sun.nio.ch.WindowsSelectorImpl$SubSelector.access$400(Unknown Source)
           at sun.nio.ch.WindowsSelectorImpl.doSelect(Unknown Source)
           at sun.nio.ch.SelectorImpl.lockAndDoSelect(Unknown Source)
           -  locked io.netty.channel.nio.SelectedSelectionKeySet@2d5dcc30
           -  locked java.util.Collections$UnmodifiableSet@388643a2
           -  locked sun.nio.ch.WindowsSelectorImpl@41be7950
           at sun.nio.ch.SelectorImpl.select(Unknown Source)
           at io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:62)
           at io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:732)
           ...
           
           
           "DestroyJavaVM" Id=21 RUNNABLE
           
           
           "Server console handler" Id=20 RUNNABLE (in native)
           at java.io.FileInputStream.readBytes(Native Method)
           at java.io.FileInputStream.read(Unknown Source)
           at java.io.BufferedInputStream.read1(Unknown Source)
           at java.io.BufferedInputStream.read(Unknown Source)
           -  locked java.io.BufferedInputStream@7526af7f
           at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
           at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
           at sun.nio.cs.StreamDecoder.read(Unknown Source)
           -  locked java.io.InputStreamReader@3659c1a7
           at java.io.InputStreamReader.read(Unknown Source)
           ...
           
           
           "Server thread" Id=18 RUNNABLE
           at net.minecraft.world.World.func_175624_G(World.java:3208)
           at net.minecraft.world.chunk.Chunk.func_186032_a(Chunk.java:472)
           at net.minecraft.world.chunk.Chunk.func_177435_g(Chunk.java:467)
           at net.minecraft.world.World.func_180495_p(World.java:912)
           at jackyy.dimensionaledibles.util.TeleporterHandler.scanColumn(TeleporterHandler.java:182)
           at jackyy.dimensionaledibles.util.TeleporterHandler.getValidYSpawnPos(TeleporterHandler.java:163)
           at jackyy.dimensionaledibles.util.TeleporterHandler.getDimPos(TeleporterHandler.java:150)
           at jackyy.dimensionaledibles.block.BlockCakeBase.calculateCoordinates(BlockCakeBase.java:304)
           ...
           
           
           "Server Infinisleeper" Id=17 TIMED_WAITING
           at java.lang.Thread.sleep(Native Method)
           at net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:82)
           
           
           "Snooper Timer" Id=16 TIMED_WAITING on java.util.TaskQueue@3cc81f9d
           at java.lang.Object.wait(Native Method)
           -  waiting on java.util.TaskQueue@3cc81f9d
           at java.util.TimerThread.mainLoop(Unknown Source)
           at java.util.TimerThread.run(Unknown Source)
           
           
           "Attach Listener" Id=5 RUNNABLE
           
           
           "Signal Dispatcher" Id=4 RUNNABLE
           
           
           "Finalizer" Id=3 WAITING on java.lang.ref.ReferenceQueue$Lock@e0e85ad
           at java.lang.Object.wait(Native Method)
           -  waiting on java.lang.ref.ReferenceQueue$Lock@e0e85ad
           at java.lang.ref.ReferenceQueue.remove(Unknown Source)
           at java.lang.ref.ReferenceQueue.remove(Unknown Source)
           at java.lang.ref.Finalizer$FinalizerThread.run(Unknown Source)
           
           
           "Reference Handler" Id=2 WAITING on java.lang.ref.Reference$Lock@4e938e44
           at java.lang.Object.wait(Native Method)
           -  waiting on java.lang.ref.Reference$Lock@4e938e44
           at java.lang.Object.wait(Unknown Source)
           at java.lang.ref.Reference.tryHandlePending(Unknown Source)
           at java.lang.ref.Reference$ReferenceHandler.run(Unknown Source)

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 11 (amd64) version 10.0
  Java Version: 1.8.0_361, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 1010972048 bytes (964 MB) / 2001207296 bytes (1908 MB) up to 2001207296 bytes (1908 MB)
  JVM Flags: 2 total; -Xms2048M -Xmx2048M
  IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
  FML: MCP 9.42 LoliASM 5.6 Powered by Forge 14.23.5.2860 146 mods loaded, 146 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State  | ID                                | Version                             | Source                                                      | Signature                                |
       |:------ |:--------------------------------- |:----------------------------------- |:----------------------------------------------------------- |:---------------------------------------- |
       | LCHIJA | minecraft                         | 1.12.2                              | minecraft.jar                                               | None                                     |
       | LCHIJA | mcp                               | 9.42                                | minecraft.jar                                               | None                                     |
       | LCHIJA | bansoukou                         | 4.3.1                               | minecraft.jar                                               | None                                     |
       | LCHIJA | FML                               | 8.0.99.99                           | forge-1.12.2-14.23.5.2860.jar                               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | forge                             | 14.23.5.2860                        | forge-1.12.2-14.23.5.2860.jar                               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJA | advancedrocketrycore              | 1                                   | minecraft.jar                                               | None                                     |
       | LCHIJA | creativecoredummy                 | 1.0.0                               | minecraft.jar                                               | None                                     |
       | LCHIJA | littletilescore                   | 1.0.0                               | minecraft.jar                                               | None                                     |
       | LCHIJA | mixinbooter                       | 4.2                                 | minecraft.jar                                               | None                                     |
       | LCHIJA | foamfixcore                       | 7.7.4                               | minecraft.jar                                               | None                                     |
       | LCHIJA | randompatches                     | 1.12.2-1.22.1.10                    | randompatches-1.12.2-1.22.1.10.jar                          | None                                     |
       | LCHIJA | fastbench                         | 1.7.3                               | FastWorkbench-1.12.2-1.7.3.jar                              | None                                     |
       | LCHIJA | actuallyadditions                 | 1.12.2-r152                         | ActuallyAdditions-1.12.2-r152.jar                           | None                                     |
       | LCHIJA | baubles                           | 1.5.2                               | Baubles-1.12-1.5.2.jar                                      | None                                     |
       | LCHIJA | actuallybaubles                   | 1.1                                 | ActuallyBaubles-1.12-1.1.jar                                | None                                     |
       | LCHIJA | redstoneflux                      | 2.1.1                               | RedstoneFlux-1.12-2.1.1.1-universal.jar                     | None                                     |
       | LCHIJA | cofhcore                          | 4.6.6                               | CoFHCore-1.12.2-4.6.6.1-universal.jar                       | None                                     |
       | LCHIJA | libvulpes                         | 0.4.2.-75                           | LibVulpes-1.12.2-0.4.2-75-universal.jar                     | None                                     |
       | LCHIJA | advancedrocketry                  | 1.7.0.-232                          | AdvancedRocketry-1.12.2-1.7.0-232-universal.jar             | None                                     |
       | LCHIJA | advsolars                         | 1.0                                 | advsolars-1.0.jar                                           | None                                     |
       | LCHIJA | appliedenergistics2               | rv6-stable-7-extended_life-v0.54.19 | appliedenergistics2-rv6-stable-7-extended_life-v0.54.19.jar | None                                     |
       | LCHIJA | bdlib                             | 1.14.3.12                           | bdlib-1.14.3.12-mc1.12.2.jar                                | None                                     |
       | LCHIJA | ae2stuff                          | 0.7.0.4                             | ae2stuff-0.7.0.4-mc1.12.2.jar                               | None                                     |
       | LCHIJA | endercore                         | 1.12.2-0.5.76                       | EnderCore-1.12.2-0.5.76.jar                                 | None                                     |
       | LCHIJA | crafttweaker                      | 4.1.20                              | CraftTweaker2-1.12-4.1.20.673.jar                           | None                                     |
       | LCHIJA | mtlib                             | 3.0.7                               | MTLib-3.0.7.jar                                             | None                                     |
       | LCHIJA | modtweaker                        | 4.0.19                              | modtweaker-4.0.20.5.jar                                     | None                                     |
       | LCHIJA | jei                               | 4.23.0                              | HadEnoughItems_1.12.2-4.23.0.jar                            | None                                     |
       | LCHIJA | codechickenlib                    | 3.2.3.358                           | CodeChickenLib-1.12.2-3.2.3.358-universal.jar               | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | brandonscore                      | 2.4.20                              | BrandonsCore-1.12.2-2.4.20.162-universal.jar                | None                                     |
       | LCHIJA | cofhworld                         | 1.4.0                               | CoFHWorld-1.12.2-1.4.0.1-universal.jar                      | None                                     |
       | LCHIJA | thermalfoundation                 | 2.6.7                               | ThermalFoundation-1.12.2-2.6.7.1-universal.jar              | None                                     |
       | LCHIJA | draconicevolution                 | 2.3.28                              | Draconic-Evolution-1.12.2-2.3.28.354-universal.jar          | None                                     |
       | LCHIJA | thermalexpansion                  | 5.5.7                               | ThermalExpansion-1.12.2-5.5.7.1-universal.jar               | None                                     |
       | LCHIJA | enderio                           | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | p455w0rdslib                      | 2.3.161                             | p455w0rdslib-1.12.2-2.3.161.jar                             | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHIJA | ae2wtlib                          | 1.0.34                              | AE2WTLib-1.12.2-1.0.34.jar                                  | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHIJA | appleskin                         | 1.0.14                              | AppleSkin-mc1.12-1.0.14.jar                                 | None                                     |
       | LCHIJA | architecturecraft                 | @VERSION@                           | architecturecraft-1.12-3.98.jar                             | None                                     |
       | LCHIJA | thedragonlib                      | 1.12.2-5.3.0                        | thedragonlib-1.12.2-5.3.0.jar                               | None                                     |
       | LCHIJA | armorplus                         | 1.12.2-11.25.1.62                   | armorplus-1.12.2-11.25.1.62.jar                             | None                                     |
       | LCHIJA | attributefix                      | 1.0.4                               | AttributeFix-1.12.2-1.0.4.jar                               | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | autoreglib                        | 1.3-32                              | AutoRegLib-1.3-32.jar                                       | None                                     |
       | LCHIJA | avaritia                          | 3.3.0                               | Avaritia-1.12.2-3.3.0.37-universal.jar                      | None                                     |
       | LCHIJA | base                              | 3.14.0                              | base-1.12.2-3.14.0.jar                                      | None                                     |
       | LCHIJA | betterquesting                    | 4.1.0                               | BetterQuestingUnofficial-4.1.0.jar                          | None                                     |
       | LCHIJA | bookshelf                         | 2.3.590                             | Bookshelf-1.12.2-2.3.590.jar                                | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | forgelin                          | 1.8.4                               | Forgelin-1.8.4.jar                                          | None                                     |
       | LCHIJA | bountiful                         | 2.2.2                               | Bountiful-2.2.2.jar                                         | None                                     |
       | LCHIJA | buildinggadgets                   | 2.8.4                               | BuildingGadgets-2.8.4.jar                                   | None                                     |
       | LCHIJA | spark                             | 1.5.2                               | spark-forge1122.jar                                         | None                                     |
       | LCHIJA | loliasm                           | 5.6                                 | censoredasm5.6.jar                                          | None                                     |
       | LCHIJA | chameleon                         | 1.12-4.1.3                          | Chameleon-1.12-4.1.3.jar                                    | None                                     |
       | LCHIJA | chesttransporter                  | 2.8.8                               | ChestTransporter-1.12.2-2.8.8.jar                           | None                                     |
       | LCHIJA | chisel                            | MC1.12.2-1.0.2.45                   | Chisel-MC1.12.2-1.0.2.45.jar                                | None                                     |
       | LCHIJA | clumps                            | 3.1.2                               | Clumps-3.1.2.jar                                            | None                                     |
       | LCHIJA | collective                        | 3.0                                 | collective-1.12.2-3.0.jar                                   | None                                     |
       | LCHIJA | contenttweaker                    | 1.12.2-4.10.0                       | ContentTweaker-1.12.2-4.10.0.jar                            | None                                     |
       | LCHIJA | craftelytra                       | 1.2                                 | CraftableElytra-1.1.jar                                     | None                                     |
       | LCHIJA | cns                               | 1.0                                 | CraftingNetherStar 1.12.1 -1.1.jar                          | None                                     |
       | LCHIJA | craftingtweaks                    | 9.0.1                               | CraftingTweaks_1.12.2-9.0.1.jar                             | None                                     |
       | LCHIJA | crafttweakerjei                   | 2.0.3                               | CraftTweaker2-1.12-4.1.20.673.jar                           | None                                     |
       | LCHIJA | creativecore                      | 1.10.0                              | CreativeCore_v1.10.69_mc1.12.2.jar                          | None                                     |
       | LCHIJA | cucumber                          | 1.1.3                               | Cucumber-1.12.2-1.1.3.jar                                   | None                                     |
       | LCHIJA | darkutils                         | 1.8.230                             | DarkUtils-1.12.2-1.8.230.jar                                | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | patchouli                         | 1.0-23.6                            | Patchouli-1.0-23.6.jar                                      | None                                     |
       | LCHIJA | deepmoblearning                   | 1.0.0                               | DeepMobEvolution-1.1.2.jar                                  | None                                     |
       | LCHIJA | defaultworldgenerator-port        | 1.12-2.3.1                          | DefaultWorldGenerator-port-1.12-2.3.1.jar                   | None                                     |
       | LCHIJA | gregtech                          | 2.5.0-beta                          | gregtech-1.12.2-2.5.0-beta.jar                              | None                                     |
       | LCHIJA | devtech                           | 2.1.2                               | devtech-2.1.2.jar                                           | None                                     |
       | LCHIJA | diethopper                        | 1.1                                 | diethopper-1.1.jar                                          | None                                     |
       | LCHIJA | difficultylock                    | 1.2                                 | difficultylock_1.12.2-1.2.jar                               | None                                     |
       | LCHIJA | theoneprobe                       | 1.4.28                              | theoneprobe-1.12-1.4.28.jar                                 | None                                     |
       | LCHIJA | dimensionaledibles                | 1.12.2-1.6.4                        | DimensionalEdibles-1.12.2-1.6.4.jar                         | None                                     |
       | LCHIJA | effortlessbuilding                | 1.12.2-2.16                         | effortlessbuilding-1.12.2-2.16.jar                          | None                                     |
       | LCHIJA | elevatorid                        | 1.3.14                              | ElevatorMod-1.12.2-1.3.14.jar                               | None                                     |
       | LCHIJA | enderiointegrationtic             | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderiobase                       | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioconduits                   | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioconduitsappliedenergistics | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioconduitsopencomputers      | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioconduitsrefinedstorage     | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderiointegrationforestry        | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderiointegrationticlate         | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioinvpanel                   | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | ftblib                            | 5.4.7.2                             | FTBLib-5.4.7.2.jar                                          | None                                     |
       | LCHIJA | enderiomachines                   | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderiopowertools                 | 5.3.70                              | EnderIO-1.12.2-5.3.70.jar                                   | None                                     |
       | LCHIJA | enderioendergy                    | 5.3.70                              | EnderIO-endergy-1.12.2-5.3.70.jar                           | None                                     |
       | LCHIJA | enderstorage                      | 2.4.6.137                           | EnderStorage-1.12.2-2.4.6.137-universal.jar                 | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | extendedcrafting                  | 1.7.0                               | ExtendedCrafting-Nomifactory-Edition-1.7.0.jar              | None                                     |
       | LCHIJA | extrautils2                       | 1.0                                 | extrautils2-1.12-1.9.9.jar                                  | None                                     |
       | LCHIJA | fasterladderclimbing              | 0.1-146                             | FasterLadderClimbing-1.12-0.1-146.jar                       | None                                     |
       | LCHIJA | fastleafdecay                     | v14                                 | FastLeafDecay-v14.jar                                       | None                                     |
       | LCHIJA | foamfix                           | 0.10.14-1.12.2                      | foamfix-0.10.14-1.12.2.jar                                  | None                                     |
       | LCHIJA | forgemultipartcbe                 | 2.6.2.83                            | ForgeMultipart-1.12.2-2.6.2.83-universal.jar                | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJA | microblockcbe                     | 2.6.2.83                            | ForgeMultipart-1.12.2-2.6.2.83-universal.jar                | None                                     |
       | LCHIJA | minecraftmultipartcbe             | 2.6.2.83                            | ForgeMultipart-1.12.2-2.6.2.83-universal.jar                | None                                     |
       | LCHIJA | forgivingvoid                     | 1.1.0                               | ForgivingVoid_1.12.2-1.1.0.jar                              | None                                     |
       | LCHIJA | storagedrawers                    | 5.2.2                               | StorageDrawers-1.12.2-5.4.2.jar                             | None                                     |
       | LCHIJA | framedcompactdrawers              | 1.2.7                               | framedcompactdrawers-1.2.7.jar                              | None                                     |
       | LCHIJA | ftbbackups                        | 1.1.0.1                             | FTBBackups-1.1.0.1.jar                                      | None                                     |
       | LCHIJA | ftbutilities                      | 5.4.1.131                           | FTBUtilities-5.4.1.131.jar                                  | None                                     |
       | LCHIJA | gcym                              | 1.1.6                               | GregicalityMultiblocks-1.12.2-1.1.6.jar                     | None                                     |
       | LCHIJA | librarianlib                      | 4.22                                | librarianlib-1.12.2-4.22.jar                                | None                                     |
       | LCHIJA | hooked                            | %VERSION%                           | hooked-1.0.3.jar                                            | None                                     |
       | LCHIJA | inventorytweaks                   | 1.63+release.109.220f184            | InventoryTweaks-1.63.jar                                    | 55d2cd4f5f0961410bf7b91ef6c6bf00a766dcbe |
       | LCHIJA | journeymap                        | 1.12.2-5.7.1                        | journeymap-1.12.2-5.7.1.jar                                 | None                                     |
       | LCHIJA | jecalculation                     | 1.12.2-3.2.7                        | JustEnoughCalculation-1.12.2-3.2.7.jar                      | None                                     |
       | LCHIJA | justenoughdimensions              | 1.6.0-dev.20200416.184714           | justenoughdimensions-1.12.2-1.6.0-dev.20200416.184714.jar   | 2b03e1423915a189b8094816baa18f239d576dff |
       | LCHIJA | littletiles                       | 1.5.0                               | LittleTiles_v1.5.57_mc1.12.2.jar                            | None                                     |
       | LCHIJA | lostcities                        | 2.0.22                              | lostcities-1.12-2.0.22.jar                                  | None                                     |
       | LCHIJA | morefurnaces                      | 1.11.0                              | MoreFurnaces-Nomifactory-Edition-1.12.2-1.11.0.jar          | None                                     |
       | LCHIJA | morelibs                          | 1.2.2                               | morelibs-1.12.2-1.2.2.jar                                   | None                                     |
       | LCHIJA | mrtjpcore                         | 2.1.4.43                            | MrTJPCore-1.12.2-2.1.4.43-universal.jar                     | None                                     |
       | LCHIJA | projectred-core                   | 4.9.4.120                           | ProjectRed-1.12.2-4.9.4.120-Base.jar                        | None                                     |
       | LCHIJA | moreplates                        | 5.5.2                               | moreplates-1.12.2-5.5.2.jar                                 | None                                     |
       | LCHIJA | morpheus                          | 1.12.2-3.5.106                      | Morpheus-1.12.2-3.5.106.jar                                 | None                                     |
       | LCHIJA | multiblocktweaker                 | 1.12.2-stable-gtceu-1.8.2           | multiblocktweaker-1.12.2-stable-gtceu-1.8.2.jar             | None                                     |
       | LCHIJA | nonetherportals                   | 1.0.0                               | nonetherPortals_v1.0.0formc1.12.2.jar                       | None                                     |
       | LCHIJA | nuclearcraft                      | 2.18y                               | NuclearCraft-2.18y-1.12.2-patched.jar                       | None                                     |
       | LCHIJA | openglider                        | @VERSION@                           | OpenGlider-1.12.1-1.1.0.jar                                 | None                                     |
       | LCHIJA | packagedauto                      | 1.12.2-1.0.5.19                     | PackagedAuto-1.12.2-1.0.5.19.jar                            | None                                     |
       | LCHIJA | packagedexcrafting                | 1.12.2-1.0.2.3                      | PackagedExCrafting-1.12.2-1.0.2.3.jar                       | None                                     |
       | LCHIJA | packmode                          | 1.2.0                               | packmode-1.12.2-1.2.0.jar                                   | None                                     |
       | LCHIJA | placebo                           | 1.6.0                               | Placebo-1.12.2-1.6.0.jar                                    | None                                     |
       | LCHIJA | projectred-integration            | 4.9.4.120                           | ProjectRed-1.12.2-4.9.4.120-integration.jar                 | None                                     |
       | LCHIJA | projectred-transmission           | 4.9.4.120                           | ProjectRed-1.12.2-4.9.4.120-integration.jar                 | None                                     |
       | LCHIJA | redstonearsenal                   | 2.6.6                               | RedstoneArsenal-1.12.2-2.6.6.1-universal.jar                | None                                     |
       | LCHIJA | simplyjetpacks                    | 2.2.15.69                           | SimplyJetpacks2-1.12.2-2.2.15.69.jar                        | None                                     |
       | LCHIJA | snad                              | 1.12.1-1.7.09.16a                   | Snad-1.12.1-1.7.09.16a.jar                                  | None                                     |
       | LCHIJA | solarflux                         | 12.4.11                             | SolarFluxReborn-1.12.2-12.4.11.jar                          | 9f5e2a811a8332a842b34f6967b7db0ac4f24856 |
       | LCHIJA | supersoundmuffler                 | 1.0.2.10                            | supersoundmuffler-revived_1.12.2_1.0.2.10.jar               | None                                     |
       | LCHIJA | thermalcultivation                | 0.3.6                               | ThermalCultivation-1.12.2-0.3.6.1-universal.jar             | None                                     |
       | LCHIJA | thermalinnovation                 | 0.3.6                               | ThermalInnovation-1.12.2-0.3.6.1-universal.jar              | None                                     |
       | LCHIJA | tombmanygraves                    | 1.12-4.2.0                          | TombManyGraves-1.12-4.2.0.jar                               | None                                     |
       | LCHIJA | topaddons                         | 1.12.2-1.13.0                       | topaddons-1.12.2-1.13.0.jar                                 | None                                     |
       | LCHIJA | torchmaster                       | 1.8.5.0                             | torchmaster_1.12.2-1.8.5.0.jar                              | None                                     |
       | LCHIJA | trashslot                         | 8.4.10                              | TrashSlot_1.12.2-8.4.10.jar                                 | None                                     |
       | LCHIJA | ulv_covers                        | 0.2.2                               | ULVCovers-1.12.2-0.2.2.jar                                  | None                                     |
       | LCHIJA | wct                               | 3.12.97                             | WirelessCraftingTerminal-1.12.2-3.12.97.jar                 | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHIJA | worleycaves                       | 1.5.2                               | worleycaves-1.12.2-1.5.2.jar                                | None                                     |
       | LCHIJA | xtones                            | 1.2.2                               | Xtones-1.2.2.jar                                            | None                                     |
       | LCHIJA | densemetals                       | 2.0.0                               | DenseMetals-1.12.2-2.0.0.30.jar                             | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJA | phosphor-lighting                 | 1.12.2-0.2.6                        | phosphor-1.12.2-0.2.6+build50-universal.jar                 | f0387d288626cc2d937daa504e74af570c52a2f1 |
       | LCHIJA | librarianliblate                  | 4.22                                | librarianlib-1.12.2-4.22.jar                                | None                                     |
  Loaded coremods (and transformers): RandomPatches (randompatches-1.12.2-1.22.1.10.jar)
                                        com.therandomlabs.randompatches.core.RPTransformer
                                      -Bansoukou- (_bansoukou-4.3.1.jar)
                                        
                                      RBLoadingPlugin (RealBench-1.12.2-1.3.3.jar)
                                        pw.prok.realbench.asm.RBTransformer
                                      LibrarianLib Plugin (librarianlib-1.12.2-4.22.jar)
                                        com.teamwizardry.librarianlib.asm.LibLibTransformer
                                      ForgelinPlugin (Forgelin-1.8.4.jar)
                                        
                                      GregTechLoadingPlugin (gregtech-1.12.2-2.5.0-beta.jar)
                                        gregtech.asm.GregTechTransformer
                                      Do not report to Forge! (If you haven't disabled the FoamFix coremod, try disabling it in the config! Note that this bit of text will still appear.) (foamfix-0.10.14-1.12.2.jar)
                                        pl.asie.foamfix.coremod.FoamFixTransformer
                                      LoliASM (censoredasm5.6.jar)
                                        
                                      JeiUtilitiesLoadingPlugin (JEI-Utilities-1.12.2-0.2.10.jar)
                                        com.github.vfyjxf.jeiutilities.asm.JeiUtilitiesClassTransformer
                                      AdvancedRocketryPlugin (AdvancedRocketry-1.12.2-1.7.0-232-universal.jar)
                                        zmaster587.advancedRocketry.asm.ClassTransformer
                                      MixinBooter (!mixinbooter-4.2.jar)
                                        
                                      Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
                                        invtweaks.forge.asm.ContainerTransformer
                                      EnderCorePlugin (EnderCore-1.12.2-0.5.76-core.jar)
                                        com.enderio.core.common.transform.EnderCoreTransformer
                                        com.enderio.core.common.transform.SimpleMixinPatcher
                                      PhosphorFMLLoadingPlugin (phosphor-1.12.2-0.2.6+build50-universal.jar)
                                        
                                      CreativePatchingLoader (CreativeCore_v1.10.69_mc1.12.2.jar)
                                        
                                      AE2ELCore (appliedenergistics2-rv6-stable-7-extended_life-v0.54.19.jar)
                                        appeng.core.transformer.AE2ELTransformer
                                      LittlePatchingLoader (LittleTiles_v1.5.57_mc1.12.2.jar)
                                        com.creativemd.littletiles.LittleTilesTransformer
  AE2 Version: stable rv6-stable-7-extended_life-v0.54.19 for Forge 14.23.5.2847
  Ender IO: No known problems detected.
            Authlib is : /C:/Users/May/Desktop/nomi-ceu-server-1.5/minecraft_server.1.12.2.jar
            
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
            !!!You are looking at the diagnostics information, not at the crash.       !!!
            !!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  List of loaded APIs: * actuallyadditionsapi (34) from ActuallyAdditions-1.12.2-r152.jar
                       * ae2wtlib|API (1.0.34) from AE2WTLib-1.12.2-1.0.34.jar
                       * appliedenergistics2|API (rv6) from appliedenergistics2-rv6-stable-7-extended_life-v0.54.19.jar
                       * ArmorPlusApi (2.1) from armorplus-1.12.2-11.25.1.62.jar
                       * Base|API (1.0.0) from base-1.12.2-3.14.0.jar
                       * Baubles|API (1.4.0.2) from Baubles-1.12-1.5.2.jar
                       * BetterQuesting|API (3.2) from BetterQuestingUnofficial-4.1.0.jar
                       * BetterQuesting|API2 (3.1) from BetterQuestingUnofficial-4.1.0.jar
                       * BetterWithModsAPI (Beta 0.6) from AppleSkin-mc1.12-1.0.14.jar
                       * chesttransporter|api (2) from ChestTransporter-1.12.2-2.8.8.jar
                       * Chisel-API (0.0.1) from Chisel-MC1.12.2-1.0.2.45.jar
                       * ChiselAPI|Carving (0.0.1) from Chisel-MC1.12.2-1.0.2.45.jar
                       * cofhapi (2.5.0) from CoFHCore-1.12.2-4.6.6.1-universal.jar
                       * CraftingTweaks|API (4.1) from CraftingTweaks_1.12.2-9.0.1.jar
                       * DraconicEvolution|API (1.3) from Draconic-Evolution-1.12.2-2.3.28.354-universal.jar
                       * enderioapi (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|addon (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|capacitor (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|conduits (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|farm (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|redstone (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|teleport (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|tools (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|upgrades (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * jeresources|API (0.9.2.60) from JustEnoughResources-1.12.2-0.9.2.60.jar
                       * journeymap|client-api (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-display (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-event (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-model (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-util (1.4) from journeymap-1.12.2-5.7.1.jar
                       * JustEnoughItemsAPI (4.13.0) from HadEnoughItems_1.12.2-4.23.0.jar
                       * Open Glider|API (0.1) from OpenGlider-1.12.1-1.1.0.jar
                       * PatchouliAPI (6) from Patchouli-1.0-23.6.jar
                       * projectred|api (2.1) from ProjectRed-1.12.2-4.9.4.120-Base.jar
                       * redstonefluxapi (2.1.1) from RedstoneFlux-1.12-2.1.1.1-universal.jar
                       * StorageDrawersAPI (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|event (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|registry (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|render (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|storage (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|storage-attribute (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * theoneprobe_api (1.4.4) from theoneprobe-1.12-1.4.28.jar
                       * wct|api (1.1) from WirelessCraftingTerminal-1.12.2-3.12.97.jar
  Extended Crafting: Nomifactory Edition: You are using a fork of Extended Crafting created for the Nomifactory modpack.
                                          If the error above is a NoSuchFieldError or a NoSuchMethodError relating to
                                          com.blakebr0.extendedcrafting,
                                          then please report to https://github.com/Nomifactory/ExtendedCrafting/issues
                                          with this crash report.
                                          Otherwise, you can ignore this message.
  AE2 Integration: IC2:OFF, GTCE:ON, RC:OFF, MFR:OFF, Waila:OFF, Mekanism:OFF, OpenComputers:OFF, THE_ONE_PROBE:ON, TESLA:OFF, CRAFTTWEAKER:ON
  Suspected Mods: Dimensional Edibles: Nomifactory Edition (dimensionaledibles)
  Profiler Position: N/A (disabled)
  Player Count: 1 / 20; [EntityPlayerMP['Mayamar_'/250, l='world', x=-791.41, y=84.00, z=152.09]]
  Is Modded: Definitely; Server brand changed to 'fml,forge'
  Type: Dedicated Server (map_server.txt)
```

**Comment by Exaxxion** [comment]
The resolution for #29 was implementing an entirely new feature, Island Cakes. It should not have affected the Void Cake in any way.

That said I will see if I can reproduce the issue based on your description.

**Comment by Exaxxion** [comment]
I have an idea of what happened.

Looking at the stack trace, it appears that the server process stopped itself after getting stuck in a loop for 60 seconds where we are looking for a viable spawn location, spiraling out from (0,0,0) which is the default value when configs are blank.

In the context of Nomi-CEu, this is happening in the void dimension which contains nothing in it, so it will never find a viable spawn location no matter how long it searches.

~~I'm not sure why you are deleting the coordinates, which ensures that the player spawns at a specific location (generating a 3x3 of obsidian under them).~~

**Comment by Exaxxion** [comment]
> I'm trying to make it so that players get sent to the corresponding coordinate in the void world when they use the cake, as opposed to everyone being sent to the same place, which isn't desirable on my particular server.

This is what island cakes are for. Protect the spawn location, place down an island cake for players to use if they end up on the spawn platform, then they will be sent out to their own personal "island" (or a shared island if they're in an FTB team).

The last cake you ate in a dimension is saved to your player data, and it is recalled when you eat a cake that takes you to that dimension again. Alternately you can /home your island or set up dislocators/teleporters.
